internal/providers/terraform/azure/testdata/bastion_host_test/bastion_host_test.golden

```diff
@@ -0,0 +1,13 @@
+
+ Name                                            Monthly Qty  Unit   Monthly Cost 
+                                                                                  
+ azurerm_bastion_host.example                                                     
+ ├─ Azure Bastion (Basic)                                730  hours       $138.70 
+ ├─ Outbound data transfer (first 10TB) (Basic)       10,000  GB        $1,200.00 
```

Comment:
I think we can leave the ` (Basic)` off since it looks they don't have different pricing (according to the pricing page).
```suggestion
 ├─ Outbound data transfer (first 10TB)       10,000  GB        $1,200.00 
```